src/solver/conv_asm_3x3u.cpp

```diff
@@ -209,8 +209,11 @@ bool ConvAsm3x3U::IsApplicable(const ConvolutionContext& params) const
         && (params.n_inputs / params.group_counts) % 4 == 0 /// \todo: remove restriction that (n_inputs/group_counts) must be multiple of 4
         && params.in_width > 3
         && params.in_width <= 1000
-        && IN_BUF_SZ <= 4 * GIB
-        && OUT_BUF_SZ <= 4 * GIB
+        && h_w < std::pow(2, 29)
```

Comment:
`filters_size` should be less than 4GB

Reply:
I guess you can find this check at line 219